src/js/start.js

```diff
@@ -85,7 +85,9 @@ $(function () {
     $(this).attr('tabindex', '-1');
   });
 
-  toggleFormInput($('.usa-show_password'), 'Show Password', 'Hide Password');
-  toggleFormInput($('.usa-show_multipassword'), 'Show my typing', 'Hide my typing');
-  validator($('.js-validate_password'));
+  if (/form-controls/.test(window.location.pathname)) {
```

Comment:
Ah, so, in that case, we should probably move docs specific code to the styleguide.js file, correct?  Initializing the all the plugins with their respective DOM elements wouldn't relevant to the overall package, just to the docs pages.